client/apps/game/src/ui/components/settlement/mint-village-pass-modal.tsx

```
@@ -1,190 +1,132 @@
-  // };
+    setSelectedRealm(selectedLocation);
+  }, [availableVillages, selectedCoords]);
 
+  // Update direction options when selectedRealm changes
   useEffect(() => {
-    const loadDirectionOptions = async () => {
-      if (!selectedRealm || !realmCheckResult) {
-        setDirectionOptions([]);
-        return;
-      }
+    if (selectedRealm && selectedRealm.directions_left) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)